checkov/common/bridgecrew/platform_integration.py

```diff
@@ -554,6 +586,8 @@ def persist_scan_results(self, scan_reports: list[Report]) -> None:
         dpath.merge(reduced_scan_reports, checks_metadata_paths)
         persist_checks_results(reduced_scan_reports, self.s3_client, self.bucket, self.repo_path)
 
+        self.persist_sast_checks_results(scan_reports)
```

Comment:
Make sure you strip any code from the report if `self.on_prem` is true (see https://github.com/bridgecrewio/checkov/blob/e5e2c453edc4addbc7660b770032d0af33c0f5aa/checkov/common/bridgecrew/platform_integration.py#L551 ).

Please also check that `enrich_and_persist_checks_metadata` will remove any code if it is stored in a different way than the current reports (`code_block`).